doc/before-after-reboot-checks.md

```diff
@@ -0,0 +1,46 @@
+# Before and After Reboot Checks
+
+CLUO can require custom node annotations before a node is allowed to reboot or
+before a node is allowed to become schedulable after a reboot. 
+
+## Configuring the `update-operator`
+
+When depolying the `update-operator`, you can provide a comma-separated list of
+node annotations to look for before a reboot should occur, and another one for
+after a reboot has occured. The list is provided as a command line argument to
+the binary -
+
+```bash
+update-operator --before-reboot-annotations anno1,anno2 --after-reboot-annotations anno3,anno4
```

Comment:
I think my comment here was lost.

Users will be running the `update-operator` image on Kubernetes, likely with our example deployment, let's show the flags in a manifest context:

```
        ...
        command:
        - "/bin/update-operator"
        - "--before-reboot-annotations=custom-before-reboot"
        - "--after-reboot-annotations=custom-after-reboot,"
        env:
        ...
```